### DISCUSSION TITLE
kaspa-miner-kaspa-miner-v0.2.0-GPU-0.1-win64-amd64 is failing with "a PTX  JIT compilation failed"

### DISCUSSION THREAD

**Discussion by Shekelme** [tmrlvi/kaspa-miner#7] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/26449500/147678295-85a7b11c-ea1b-4b4e-a17a-baf7e7a41ad4.png)
What version of CUDA is needed to run this miner?

**Comment by tmrlvi** [comment, marked as answer]
In Windows you need Cuda 11.5+, in Linux you need Cuda 11.2+
